helpers/css-box-shadows-generator.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "CSS Box Shadows Generator",
+  "desc": "CSS Box Shadow Generator enables shadow manipulation and control over all shadow properties. It provides instant updates to shadow codes for easy copying and individual shadow descriptions.",
```

Comment:
Could you adjust the `desc` field.

From the contribution guidelines

> the desc field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Reply:
Is this description better?

"Create, edit, and generate multiple custom CSS box shadows."

Reply:
Description Changed 👍